test/specs/elements/Container/Container-test.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+import Container from 'src/elements/Container/Container'
+import * as common from 'test/specs/commonTests'
+
+describe('Container', () => {
+  common.isConformant(Container)
+  common.rendersChildren(Container)
+  common.hasUIClassName(Container)
+
+  common.propKeyOnlyToClassName(Container, 'text')
+  common.propKeyOnlyToClassName(Container, 'left', { className: 'aligned' })
+  common.propKeyOnlyToClassName(Container, 'center', { className: 'aligned' })
+  common.propKeyOnlyToClassName(Container, 'right', { className: 'aligned' })
+  common.propKeyOnlyToClassName(Container, 'justified')
+  common.propKeyOnlyToClassName(Container, 'fluid')
+
+  it('renders a <div /> element', () => {
+    shallow(<Container />)
+      .should.have.tagName('div')
+  })
+
+  it('adds the Container class', () => {
+    shallow(<Container />)
+      .should.have.className('container')
+  })
```

Comment:
So the common.propKeyOnlyToClassName are still required, but not the 2 
'it' blocks?

> Levi Thomason <mailto:notifications@github.com>
> 24 June 2016 at 17:26
>
> In test/specs/elements/Container/Container-test.js 
> <https://github.com/TechnologyAdvice/stardust/pull/277#discussion_r68423748>:
>
> > + common.propKeyOnlyToClassName(Container, 'text')
> > + common.propKeyOnlyToClassName(Container, 'left', { className: 
> 'aligned' })
> > + common.propKeyOnlyToClassName(Container, 'center', { className: 
> 'aligned' })
> > + common.propKeyOnlyToClassName(Container, 'right', { className: 
> 'aligned' })
> > + common.propKeyOnlyToClassName(Container, 'justified')
> > + common.propKeyOnlyToClassName(Container, 'fluid')
> > +
> > + it('renders a <div /> element', () => {
> > + shallow(<Container />)
> > + .should.have.tagName('div')
> > + })
> > +
> > + it('adds the Container class', () => {
> > + shallow(<Container />)
> > + .should.have.className('container')
> > + })
>
> This test is automated as part of the |common.isConformant()| test. 
> All components are required to include their Semantic UI component 
> className.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/TechnologyAdvice/stardust/pull/277/files/a083fd536fd2f416c6214e2af929e6c51b4e8d58#r68423748>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AAWHs8T9VMTH_JMod5KlJmmO1HaBUkeJks5qPAU9gaJpZM4I9hiP>.
>

-- 
Sent from Postbox 
<https://www.postbox-inc.com/?utm_source=email&utm_medium=siglink&utm_campaign=reach>